### ISSUE TITLE
log levels

### ISSUE THREAD

**Issue by gdarcan** [OpenVidu/openvidu#243]
it should be customized like below:

var options = {
    loglevel: 1
}
var OV = new OpenVidu(options);

this will instruct openvidu to adjust the level of logging and levels can be like below (universally accepted hierarchy)

0: OFF
1: FATAL
2: ERROR
3 WARN
4: INFO
5: DEBUG
6: TRACE
7: ALL

**Comment by pabloFuente** [comment]
The idea is nice. I think the logging levels should be these, being the standard for most browsers:

```
critical -  ...............  + critical
(off)/trace/debug/info/warn/error/(all)
```

And maybe the way of configuring these logs should be by calling a method, just like advanced configuration is set like this: https://openvidu.io/api/openvidu-browser/classes/openvidu.html#setadvancedconfiguration

I think that is clearer way if users want to dynamically change the logging level on runtime.

**Comment by gdarcan** [comment]
Yes you're right about browser srandards, how to start implementing ?


Pablo Fuente Pérez <notifications@github.com>, 26 Nis 2019 Cum, 15:45
tarihinde şunu yazdı:

**Comment by pabloFuente** [comment]
Well, I think it's as simple as taking method [OpenVidu.enableProdMode](https://github.com/OpenVidu/openvidu/blob/master/openvidu-browser/src/OpenVidu/OpenVidu.ts#L460) and adapt it to behave as desired with those logging levels. Should be a very simple update
